src/tests/multihost/ipa/test_subid_ranges.py

```diff
@@ -0,0 +1,132 @@
+""" Automation of IPA bugs """
+
+import pytest
+import subprocess
+import time
+import os
+from sssd.testlib.common.utils import SSHClient
+
+
+test_password = "Secret123"
+user = 'admin'
+
+
+def execute_cmd(multihost, command):
+    """ Execute command on client """
+    cmd = multihost.client[0].run_command(command)
+    return cmd
+
+
+@pytest.mark.usefixtures('install_list_subid_ranges',
+                         'setup_ipa_client')
+@pytest.mark.tier1
+class TestSubid(object):
+    """
+    This is for ipa bugs automation
+    """
+    def test_subid_feature(self, multihost,
+                           create_bkp_for_subid_files):
+        """
+        :Title: support subid ranges managed by FreeIPA
+        :id: 50bcdc28-00c8-11ec-bef4-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1803943
+        :steps:
+            1. Generate subid for user admin
+            2. Test newuidmap command
+            3. Test newgidmap command
```

Comment:
Where do you setup nsswitch.conf::subid in this test?

Reply:
class fixture "install_list_subid_ranges" will take care it for entire class

Reply:
This fixture makes sure there is no field 'subid'.
How test is expected to work?

Reply:
IIUC, both tests should work with `subid: sss` set.

Reply:
For now there is no pre existing subid field in nsswitch . This field will be added later in the second test

Reply:
How first test is expected to work without `subid: sss`?

Reply:
That is taken cared by fixture "create_bkp_for_subid_files"

Reply:
Ok, better now, thanks.
But fixture name - `create_bkp_for_subid_files` - is misleading.
Please either rename this function to highlight it also modifies config (besides makes a backup) or split this fixture into two.

Reply:
This is not needed as these lies are already there to validate the results .

        result = execute_cmd(multihost, f"cat /proc/{proces_id}/uid_map")
        assert f'{uid_start} {uid_start + 1}' in result.stdout_text

Reply:
renamed

Reply:
What is not needed?
Name of your functions - `create_bkp_for_subid_files` - suggests this function creates backup files.
But besides this function also modifies config. I'd like this to be reflected in function name or function to be split.

Reply:
its renamed now